modules/core/src/test/scala/org/scalasteward/core/edit/EditAlgTest.scala

```diff
@@ -258,10 +258,22 @@ class EditAlgTest extends FunSuite {
       newerVersions = Nel.of("0.9.9")
     )
     val original = Map(
-      ".mill-version" -> """0.9.5"""
+      ".mill-version" -> """0.9.5""",
+      ".travis.yml" ->
+        """
+          |env
+          |  - TEST_MILL_VERSION=0.9.4
+          |  - TEST_MILL_VERSION=0.9.5
```

Comment:
This is the only relevant line for this test, right? We're checking that the version number is only updated in `.mill-version` and not here. I would therefore delete all other lines to keep it lean:
`".travis.yml" -> "- TEST_MILL_VERSION=0.9.5"`